azure-pipelines.yml

```diff
@@ -12,5 +12,15 @@ jobs:
   pool:
     vmImage: 'vs2017-win2016'
   steps:
-  - powershell: ./build.ps1
-    displayName: 'Cake Build'
\ No newline at end of file
+  - powershell: |
+      $ENV:CAKE_SKIP_GITVERSION=([string]::IsNullOrEmpty($ENV:SYSTEM_PULLREQUEST_PULLREQUESTID) -eq $False).ToString()
+      .\build.ps1
+      exit $LASTEXITCODE 
+    displayName: 'Cake Build'
+- job: Ubuntu
+  pool:
+    vmImage: 'ubuntu-16.04'
```

Comment:
Using `ubuntu-18.04` isn't still an option?

Reply:
There's currently only a Ubuntu 16.04 VM available. It's possible to build on other distros though if building in a Docker container, but this would be something for another PR. Currently I'm happy if build runs at least on one Linux distro to prove that it basically works there.

Reply:
Yes, seems to take some more time :)

https://github.com/Microsoft/azure-pipelines-image-generation/issues/506